setup.py

```diff
@@ -149,179 +134,23 @@ def append(self, item):
 else:
     for path in pkg_config_cflags:
         assert path.find(CFLAGS_INCLUDE_PATH_SWITCH) == 0
-        include_dirnames.add_dir(path.lstrip(CFLAGS_INCLUDE_PATH_SWITCH))
+        include_dirnames.append(path.lstrip(CFLAGS_INCLUDE_PATH_SWITCH))
```

Comment:
Please change use of `str.lstrip()` to `str.partition()[2]` and add a comment that for Python >=3.9 we should use `str.removeprefix()` instead.